crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -91,11 +93,23 @@ impl KeypairOutput {
     }
 }
 
+fn init_logging() {
+    let builder = Subscriber::default();
+
+    // set default log to info if the RUST_LOG is not set.
+    #[cfg(feature = "env-filter")]
+    {
+        let env_filter = tracing_subscriber::EnvFilter::builder()
+            .with_default_directive(LevelFilter::Info.into());
+        let builder = builder.with_env_filter(crate::EnvFilter::from_default_env());
```

Comment:
This is a new local variable, it has no impact on `builder.init()` that is called below

Reply:
oops! missed that. good catch